app/views/adventure_template_two.blade.php

```diff
@@ -0,0 +1,18 @@
+<html>
```

Comment:
go ahead and delete the whole view for this, we're only gonna have one adventure template